Parsers/ASimFileEvent/ProductParsers/FileEventMicrosoftSysmonFileCreated.yaml

```diff
@@ -28,7 +28,8 @@ ParserQuery: |
                   '</Data><Data Name="Image">'Image:string /// Image is the full path 
                   '</Data><Data Name="TargetFilename">'TargetFilename:string //// Full Path
                   '</Data><Data Name="CreationUtcTime">'CreationUtcTime:datetime  
-                  '</Data></EventData></DataItem>'
+                  '</Data>'new_ver_data '</EventData></DataItem>'
+          | parse new_ver_data with '<Data Name="User">'ActorUsername'</Data>' // parsing will work only for newer versions of sysmon -> for older will remain empty field
```

Comment:
Now, this looks fine, however, you don't need new_ver_data. Just parse again the entire string. It would be more efficient.